src/asset/js/showList.js

```diff
@@ -1,34 +1,31 @@
 /** @format */
-import { todoTask } from "./add.js";
-
-const todoListContainer = document.querySelector("#todo-list-container");
+/* eslint-disable */
+import setItem from './storage.js';
+import todoTask from '../../index.js';
+/* eslint-disable */
```

Comment:
- [ ] as the previous CR already mentioned please do not disable Eslint it will help check for your Javascript code standards. Kindly enable Eslint and fix the errors manually.